### ISSUE TITLE
✅ 🔗 🐞 Bug Report: Errors using an ACF Image as a background for elements within a Loop Grid template (Related to #22368) - [ED-12415]

### ISSUE THREAD

**Issue by micmiller-apptio** [elementor/elementor#21489]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I see an error message on the site front end using a Loop Grid if my Loop Item Template has an element that uses a dynamic tag with an ACF image field for a background image.

`
Notice: Trying to access array offset on value of type bool in /code/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-image.php on line 59 
`

After making any edit to the loop item template, I don't see the error, but the image won't show for the first item in the loop. Refreshing the page will yield the error before the first item for every subsequent page load.

### Steps to reproduce

1. Create a custom post type (I'm using Custom Post Type UI to do this)
2. Create an ACF field group with an image field and assign it to your post type
3. Create two posts of the new type and assign an image for each to use as a background
4. Create a loop item template with a spacer widget and use a dynamic tag to assign the ACF field as the background image source for either the spacer widget or the column/section containing it. It doesn't seem to matter which widget you use.
5. Create a new page with a Loop Grid using the new item template and the new post type as the query source.
6. View the page on the front end. On the first view, the background image for the first loop item will not show, the second one will
7. Refresh the page, you should see the error message in the first grid slot, the first post in the second slot and the second post in the third slot

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.21.6
	MySQL version: MariaDB Server v10.4.25
	PHP Version: 7.4.30
	PHP Memory Limit: 512M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://sandbox3-apptio-com.pantheonsite.io
	Home URL: https://sandbox3-apptio-com.pantheonsite.io
	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /blog/%postname%/
	Language: en-US
	Timezone: America/Los_Angeles
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 6.0.7
		Author: WP Engine

	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Custom Post Type Permalinks
		Version: 3.4.5
		Author: Toro_Unit

	Custom Post Type UI
		Version: 1.13.4
		Author: WebDevStudios

	Elementor
		Version: 3.11.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.11.3
		Author: Elementor.com

	Native PHP Sessions for WordPress
		Version: 1.3.4
		Author: Pantheon

	Pantheon Advanced Page Cache
		Version: 1.2.0
		Author: Pantheon

	Pantheon HUD
		Version: 0.4.2
		Author: Pantheon

	Safe SVG
		Version: 2.0.3
		Author: 10up

	Style Kits for Elementor
		Version: 2.0.1
		Author: AnalogWP

	Style Kits Pro
		Version: 2.0.1
		Author: AnalogWP

	WP Redis
		Version: 1.3.2
		Author: Pantheon, Josh Koenig, Matthew Boynes, Daniel Bachhuber, Alley Interactive

	WPS Hide Login
		Version: 1.9.8
		Author: WPServeur, NicolasKulka, wpformation


== Must-Use Plugins ==
	Apptio Custom Security Headers
		Version: 1.0
		Author: Apptio

	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

	Pantheon MU Plugin Loader
		Version: 1.0
		Author: Pantheon Systems



== Settings ==
	
	cpt_support: post, page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1


== Features ==
	Custom Fonts: 2
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Inactive by default
	Improved Asset Loading: Inactive by default
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Inactive by default
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive by default
	Editor V2: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Inactive
	Scroll Snap: Active by default
	Mega Menu: Inactive by default


== Log ==
	
PHP: 2023-02-28 14:59:21 [notice X 48][/code/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-image.php::59] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-28 15:37:01 [notice X 3][/code/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-image.php::40] Undefined offset: 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	Style Kits for Elementor: Compatible
	Style Kits Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	
	Style Kits for Elementor: Compatible
	Style Kits Pro: Compatible
				
````

</details>

**Comment by nicholaszein** [comment]
Hello @micmiller-apptio! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by hitsgee** [comment]
I have this issue aswell

**Comment by dotboylad** [comment]
My issue is slightly different:
– I have a custom post type created in ACF that has an image field.
– I have a single master for the custom post type created in Theme builder.
– I'm calling the post's ACF image for the background.
– It works in the preview of the Theme builder editor.
– It doesn't work on the front end.

Thanks in advance for help...

**Comment by nicholaszein** [comment]
Hello everyone!

Thank you for your patience! 🙏 

This bug is being worked on, and our team is looking for a solution. Hopefully, we can find it soon. 

### 🔗 As this is the same issue as #22368, I am closing comments here so we can have a centralized place to discuss and update you.

Feel free to continue the conversation there. 😉 

Kind regards

**Comment by nicholaszein** [comment]
Hello everyone!

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.19.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
